health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MobileChecklist.js

```
@@ -12,23 +12,43 @@ const MobileChecklist = ({ questions, checklistRole , typeOfChecklist}) => {
 
       <div className="mobile-container">
         <div className="mobile-header">
-          <h2 className='mobile-checklist-highlight'>{typeOfChecklist}</h2>
-          <h1 className="mobile-checklist-highlight">{checklistRole}</h1>
-          <p className="mobile-description">
-          </p>
+          <h2 className='mobile-checklist-highlight'>{typeOfChecklist} {checklistRole}</h2>
+          {/* <h1 className="mobile-checklist-highlight">{checklistRole}</h1> */}
+          <p className="mobile-description"></p>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)